lib/database_cleaner/safeguard.rb

```diff
@@ -12,8 +12,35 @@ def initialize(env)
           super("ENV['#{env}'] is set to production. Please refer to https://github.com/DatabaseCleaner/database_cleaner#safeguards")
         end
       end
+
+      class NotWhitelistedUrl < Error
+        def initialize
+          super("ENV['DATABASE_URL'] is set to a URL that is not on the whitelist. Please refer to https://github.com/DatabaseCleaner/database_cleaner#safeguards")
+        end
+      end
     end
 
+    class WhitelistedUrl
+      def run
+        raise Error::NotWhitelistedUrl if !skip? && given?
```

Comment:
@benclewis Could you refactor this to avoid the double negative? It's hard to read the combination of `!skip?` and `!DatabaseCleaner.url_whitelist`

Maybe you can make it a little more straightforward like this: 

```
def run
  return if skip?
  raise Error::NotWhitelistedUrl if database_url_not_whitelisted?
end
```